From 8d7144126a68e93f15e1d8fe634978be243d2cda Mon Sep 17 00:00:00 2001
From: Xulin Sun <xulin.sun@windriver.com>
Date: Sat, 27 Jun 2020 12:20:37 +0800
Subject: [PATCH 3/8] crypto: caam - drop unused label out_put_dev

To fix below warning:
drivers/crypto/caam/caamalg.c:3671:1: warning: label 'out_put_dev' defined but not used [-Wunused-label]

3671 | out_put_dev:

      | ^~~~~~~~~~~

Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/crypto/caam/caamalg.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/crypto/caam/caamalg.c b/drivers/crypto/caam/caamalg.c
index 2b14623cc9a8..5e45d75c3064 100644
--- a/drivers/crypto/caam/caamalg.c
+++ b/drivers/crypto/caam/caamalg.c
@@ -3668,7 +3668,6 @@ int caam_algapi_init(struct device *ctrldev)
 	if (registered)
 		pr_info("caam algorithms registered in /proc/crypto\n");
 
-out_put_dev:
 	put_device(ctrldev);
 	return err;
 }
-- 
2.17.1

